rAthena.sln

```diff
@@ -1,8 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.40629.0
-MinimumVisualStudioVersion = 12.0.40629.0
+# Visual Studio 15
+VisualStudioVersion = 15.0.26228.4
+MinimumVisualStudioVersion = 15.0.26228.4
```

Comment:
```suggestion
MinimumVisualStudioVersion = 15.0.0.0
```
Since your and my sources for that are ambigous and the check in MSVS is done with >= I would say we simply zero out after the major release number.
I tried that locally and it works perfectly fine.

I also tried what happens if I set it to 18.0.0.0 for example.
So this would be the error anyone < MSVS 2017 would get after we merge this:
![grafik](https://user-images.githubusercontent.com/3517879/218265331-cd36e095-388f-4c0a-bd9c-20b3e22400dc.png)